plugins/org.jkiss.dbeaver.ext.dm/src/org/jkiss/dbeaver/ext/dm/model/DMMetaModel.java

```diff
@@ -0,0 +1,157 @@
+        return true;
+    }
+
+    @Override
+    public JDBCStatement prepareSequencesLoadStatement(JDBCSession session, GenericStructContainer container) throws SQLException {
+        JDBCPreparedStatement dbStat = session.prepareStatement(
+                "SELECT SEQ_OBJ_INNER.NAME,SEQ_OBJ_INNER.INFO1,SEQ_OBJ_INNER.INFO3,SEQ_OBJ_INNER.INFO4 " +
+                        "FROM " +
+                        "SYSOBJECTS SEQ_OBJ_INNER, " +
+                        "SYSOBJECTS SCH_OBJ_INNER " +
+                        "WHERE " +
+                        "SEQ_OBJ_INNER.SUBTYPE$ = 'SEQ' " +
+                        "AND SCH_OBJ_INNER.ID = SEQ_OBJ_INNER.SCHID " +
+                        "AND SCH_OBJ_INNER.name = ?");
+        dbStat.setString(1, container.getName());
+        return dbStat;
+    }
+
+    @Override
+    public GenericSequence createSequenceImpl(JDBCSession session, GenericStructContainer container, JDBCResultSet dbResult) throws DBException {
+        String name = JDBCUtils.safeGetString(dbResult, 1);
+        if (CommonUtils.isEmpty(name)) {
+            return null;
+        }
+        Number lastValue = JDBCUtils.safeGetBigDecimal(dbResult, 3);
+        Number incrementBy = JDBCUtils.safeGetBigDecimal(dbResult, 4);
+        return new GenericSequence(container, name, "", lastValue, null, null, incrementBy);
+    }
+
+    @Override
+    public boolean supportsDatabaseTriggers(GenericDataSource dataSource) {
+        return true;
+    }
+
+    @Override
+    public boolean supportsTriggers(@NotNull GenericDataSource dataSource) {
+        return true;
+    }
+
+
+    @Override
+    public JDBCStatement prepareTableTriggersLoadStatement(JDBCSession session, GenericStructContainer container, GenericTableBase table) throws SQLException {
+        JDBCPreparedStatement dbStat = session.prepareStatement("SELECT " +
+                "TABTRIG_OBJ_INNER.NAME " +
+                "FROM " +
+                "SYSOBJECTS TABTRIG_OBJ_INNER, " +
+                "SYSOBJECTS SCH_OBJ_INNER " +
+                "WHERE " +
+                "TABTRIG_OBJ_INNER.SUBTYPE$ = 'TRIG' " +
+                "AND TABTRIG_OBJ_INNER.SCHID = SCH_OBJ_INNER.ID " +
+                "AND SCH_OBJ_INNER.NAME = ? ");
+        dbStat.setString(1, container.getName());
+        return dbStat;
+    }
+
+    @Override
+    public JDBCStatement prepareContainerTriggersLoadStatement(JDBCSession session, GenericStructContainer forParent) throws SQLException {
+        return super.prepareContainerTriggersLoadStatement(session, forParent);
+    }
+
+    @Override
+    public String getViewDDL(DBRProgressMonitor monitor, GenericView sourceObject, Map<String, Object> options) throws DBException {
+        try (JDBCSession session = DBUtils.openMetaSession(monitor, sourceObject, "Read view definition")) {
+            return JDBCUtils.queryString(session, "SELECT DBMS_METADATA.GET_DDL('VIEW',?,?);", sourceObject.getName(), sourceObject.getParentObject().getName());
```

Comment:
Will it work with tables? Or triggers? Maybe it is better to implement it for getTableDDL and getTriggerDDL too? As a base method but with the object type changing. 
Like it implemented here:
https://github.com/dbeaver/dbeaver/blob/2a7b33fa7eb8a6e357421d22564eaad6cbe67099/plugins/org.jkiss.dbeaver.ext.oracle/src/org/jkiss/dbeaver/ext/oracle/model/OracleUtils.java#L117